demo-shell/src/app/components/process-service/process-attachments.component.css

```diff
@@ -1,3 +1,9 @@
 adf-create-process-attachment ::ng-deep button {
```

Comment:
should be possible to get rid of this "::ng-deep" if using ViewEncapsulation.None

Reply:
done